Unable to use codex account

[ERROR] [400]: Invalid 'input[65].name': empty string. Expected a string with minimum length 1, but got an empty string instead.
❌ codex [400]: [400]: Invalid 'input[65].name': empty string. Expected a string with minimum length 1, but got an e
[15:13:06] 🔍 [AUTH] codex | total connections: 3, excludeIds: 04ebf2b6-4a8a-404f-a34a-fe6e5a704f8e,d6aa723f-f5bb-480c-ad4c-d72d2f7bc410,e93c0b39-d03a-471a-8dea-12f68b60bede, model: gpt-5.4

[15:24:44] [PENDING] END (ERROR) | provider=codex | model=gpt-5.4
[ERROR] [400]: Invalid schema for function 'mcp_dart-mcp-server_analyze_files': In context=(), object schema missing properties.
❌ codex [400]: [400]: Invalid schema for function 'mcp_dart-mcp-server_analyze_files': In context=(), object schema
[15:24:44] 🔍 [AUTH] codex | total connections: 3, excludeIds: 04ebf2b6-4a8a-404f-a34a-fe6e5a704f8e,d6aa723f-f5bb-480c-ad4c-d72d2f7bc410,e93c0b39-d03a-471a-8dea-12f68b60bede, model: gpt-5.4

this is antigravity mitm

--- a/open-sse/translator/helpers/responsesApiHelper.js
+++ b/open-sse/translator/helpers/responsesApiHelper.js
@@ -87,6 +87,8 @@ export function convertResponsesApiFormat(body) {
           tool_calls: []
         };
       }
+      // Skip items with empty/missing name — upstream APIs reject nameless tool calls (#444)
+      if (!item.name || typeof item.name !== "string" || item.name.trim() === "") continue;
       currentAssistantMsg.tool_calls.push({
         id: item.call_id,
         type: "function",

--- a/open-sse/translator/request/openai-responses.js
+++ b/open-sse/translator/request/openai-responses.js
@@ -75,6 +75,8 @@ export function openaiResponsesToOpenAIRequest(model, body, stream, credentials)
           tool_calls: []
         };
       }
+      // Skip items with empty/missing name — Codex/OpenAI reject nameless tool calls (#444)
+      if (!item.name || typeof item.name !== "string" || item.name.trim() === "") continue;
       currentAssistantMsg.tool_calls.push({
         id: item.call_id,
         type: "function",

